### ISSUE TITLE
command-line plugin does not display the prompt (spans outside of their container) 

### ISSUE THREAD

**Issue by AymDev** [PrismJS/prism#2289]
**Information:**
- Prism version: 1.20.0
- Plugins: line-numbers, autolinker, show-language, keep-markup, command-line
- Environment: Webpack Encore (a Symfony project)

**Description**
The plugin is activated (detects output lines) but it does not display the user and host. It shows on page load (< 1 sec) and then gets removed from the DOM.
I added the `keep-markup` plugin too but this time, the `<span>` elements are placed outside the `<span class="command-line-prompt"></span>` which seems to be the container according to the [example page](https://prismjs.com/plugins/command-line/) I inspected.

I use Webpack Encore, with the Babel plugin as recommended. Before calling Prism's highlight function I add the attributes on the `<pre>` (the HTML content is generated from MarkDown stored in the database). Here is my full script:
```js
// Slide content container
let content = document.getElementById('slide_content');

// Add classes for plugins
content.querySelectorAll('pre').forEach(function (pre) {

    // Display command line output
    let code = pre.querySelector(':scope > code');
    if (
        code.classList.contains('language-shell')
        || code.classList.contains('language-shell-session')
        || code.classList.contains('language-bash')
        || code.classList.contains('language-git')
    ) {
        pre.classList.add('command-line');
        pre.dataset.user = 'aymdev';
        pre.dataset.host = 'local';
        pre.dataset.filterOutput = '[output]';

    } else {

        // Or add line numbers
        pre.classList.add('line-numbers');
    }
});

// Plugins assets
require('prismjs/plugins/command-line/prism-command-line.css');

// Prism JS (configuration with Babel plugin)
import Prism from 'prismjs';
Prism.highlightAllUnder(content);
```
Removing all other plugins had no effect. No idea if this could be related to #1456 

**Temporary workaround:**
To fix this I added this at the bottom of my script:
```js
window.addEventListener('DOMContentLoaded', function () {
    content.querySelectorAll('pre.command-line > code').forEach(function (codeElement) {
        let prompt = codeElement.querySelector('span.command-line-prompt');

        // Starts at 1 as the first element is the prompt
        for (let i = 1; i < codeElement.children.length; i++) {
            // Move spans inside the prompt
            if (codeElement.children[i].getAttribute('data-user') || codeElement.children[i].attributes.length === 0) {
                prompt.insertBefore(codeElement.children[i], null);
                i--;
            } else {
                // Stop as spans to move are placed before highlighted tokens
                return;
            }
        }
    });
});
```

**Comment by RunDevelopment** [comment]
This is just a guess but could it be that you calling `Prism.highlightAllUnder(content);` is on top of Prism's automatic highlighting?

When playing around with the [Command line plugin page](https://prismjs.com/plugins/command-line/), I noticed that highlighting an element with a command prompt again, will result in the prompt being removed. 
To reproduce, just go to the plugin page, open the console, and enter `Prism.highlightAll()` repeatedly. You will see the command prompt appear and disappear.

I'll investigate.

**Comment by AymDev** [comment]
Oh, does it mean that the `import Prism from 'prismjs';` does automatic highlighting and my call to `Prism.highlightAllUnder(content);` *re-*highlights ?

Well, GitHub Pages is under maintenance but I just tried on my own page and I observed what you said.
By removing `Prism.highlightAllUnder(content);` and my workaround, it works fine. However, how can I disable automatic highlighting to keep calling `Prism.highlightAllUnder(content);` ? I'm sorry if this is a basic JS question, I'm more of a backend dev.
